README.md

```diff
@@ -1,10 +1,10 @@
 # HPE OneView SDK for GoLang
 
 ## Build Status 
-|      OV Version |  8.20|  8.10|   8.00|  7.20|   7.10|   7.00 |   6.60 |   6.50 |  6.40 | 6.30 |  6.20 | 6.10 | 6.00 |
-| --------------:| --------------:| --------------: |--------------: |--------------: |  -----------: |-----------: | ------------: | -------------: |-----------: | ----------: |-----------: |-----------: | -----------: |
-| SDK Version/Tag |  [v8.2.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v8.2.0)|  [v8.1.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v8.1.0) |  [v8.0.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v8.0.0) |  [v7.2.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v7.2.0) |  [v7.1.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v7.1.0) |   [v7.0.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v7.0.0)|   [v6.6.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.6.0)|   [v6.5.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.5.0) |  [v6.4.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.4.0) |  [v6.3.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.3.0) |  [v6.2.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.2.0) |  [v6.1.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.1.0) | [v6.0.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.0.0) |
```

Comment:
There is an issue here with the 8.3 link. Please check in UI once.

Reply:
Done

Reply:
done